homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -71,6 +75,86 @@ def validate_language(data: dict[str, Any]) -> Any:
 SAVE_DELAY = 10
 
 
+async def _async_create_default_pipeline(
+    hass: HomeAssistant, pipeline_store: PipelineStorageCollection
+) -> None:
+    """Create a default pipeline.
+
+    Use cloud for stt / tts if there's a subscription, otherwise create a text
+    only pipeline.
+    """
+    conversation_language = "en"
+    pipeline_language = "en"
+    pipeline_name = "Home Assistant"
```

Comment:
We should change this to Home Assistant Cloud if it's using cloud for `stt` and `tts`.